### PR TITLE
refact(client): adapt import & style with latest code

### PR DESCRIPTION
**Pull request by imbajin**
check all class in `client` module manually

and the basic rule is we should avoid using a **special align style**, it's not friendly for normal users to format the code (Greatly reduces the efficiency of commit)

use the config file to format the file automatically is more sensible, and we could reduce to review the `space/align/empty line/wrong import` also.

### CODE REVIEW

File: hugegraph-client/src/main/java/org/apache/hugegraph/structure/graph/Edge.java [hunk 1]
```diff
@@ -71,8 +72,8 @@ public Object sourceId() {
         }
         if (this.sourceId == null) {
             throw new InvalidOperationException(
-                      "Must set source vertex id or add vertices " +
-                      "before add edges");
+                    "Must set source vertex id or add vertices " +
```
Comment (javeme) [inline hugegraph-client/src/main/java/org/apache/hugegraph/structure/graph/Edge.java:75]: please check

File: hugegraph-client/src/test/java/org/apache/hugegraph/api/ApiTestSuite.java [hunk 2]
```diff
@@ -52,51 +52,51 @@
 
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-    PropertyKeyApiTest.class,
-    VertexLabelApiTest.class,
-    EdgeLabelApiTest.class,
-    IndexLabelApiTest.class,
-    SchemaApiTest.class,
+        PropertyKeyApiTest.class,
```
Comment (javeme) [inline hugegraph-client/src/test/java/org/apache/hugegraph/api/ApiTestSuite.java:55]: please check

File: hugegraph-client/src/test/java/org/apache/hugegraph/api/EdgeApiTest.java [hunk 3]
```diff
@@ -616,7 +616,6 @@ public void testBatchCreateWithInvalidVertexLabelAndCheck() {
 //            edgeAPI.get(ids.get(1));
 //        });
 //    }
-
```
Comment (javeme) [inline hugegraph-client/src/test/java/org/apache/hugegraph/api/EdgeApiTest.java:619]: please check

File: hugegraph-client/src/test/java/org/apache/hugegraph/api/TaskApiTest.java [hunk 4]
```diff
@@ -212,8 +212,8 @@ public void testCancel() {
         schema().vertexLabel("man").useAutomaticId().ifNotExist().create();
 
         String groovy = "for (int i = 0; i < 10; i++) {" +
-                            "hugegraph.addVertex(T.label, 'man');" +
-                            "hugegraph.tx().commit();" +
+                        "hugegraph.addVertex(T.label, 'man');" +
```
Comment (javeme) [inline hugegraph-client/src/test/java/org/apache/hugegraph/api/TaskApiTest.java:215]: prefer to add some spaces after '"'

File: hugegraph-client/src/test/java/org/apache/hugegraph/api/TaskApiTest.java [hunk 5]
```diff
@@ -233,13 +233,13 @@ public void testCancel() {
...
          */
         groovy = "for (int i = 0; i < 10; i++) {" +
-                     "hugegraph.addVertex(T.label, 'man');" +
-                     "hugegraph.tx().commit();" +
-                     "try {" +
-                         "sleep(1000);" +
-                     "} catch (InterruptedException e) {" +
-                         "break;" +
-                     "}" +
+                 "hugegraph.addVertex(T.label, 'man');" +
```
Comment (javeme) [inline hugegraph-client/src/test/java/org/apache/hugegraph/api/TaskApiTest.java:236]: ditto

File: hugegraph-client/src/test/java/org/apache/hugegraph/functional/FuncTestSuite.java [hunk 6]
```diff
@@ -24,20 +24,20 @@
...
-    VertexTest.class,
-    EdgeTest.class,
-    BatchInsertTest.class,
-    GraphManagerTest.class,
-    AuthManagerTest.class,
-    TraverserManagerTest.class,
-    MetricsManagerTest.class,
-    HugeClientHttpsTest.class,
-    HugeClientTest.class
+        PropertyKeyTest.class,
```
Comment (javeme) [inline hugegraph-client/src/test/java/org/apache/hugegraph/functional/FuncTestSuite.java:27]: ditto

File: hugegraph-client/src/test/java/org/apache/hugegraph/api/ApiTestSuite.java (hunk 2 above)
Comment (imbajin) [reply hugegraph-client/src/test/java/org/apache/hugegraph/api/ApiTestSuite.java:55]: how many spaces shall we keep in `@Annotation({p1, p2, p3})` ?  Is 4 spaces right?

File: hugegraph-client/src/test/java/org/apache/hugegraph/functional/FuncTestSuite.java (hunk 6 above)
Comment (imbajin) [reply hugegraph-client/src/test/java/org/apache/hugegraph/functional/FuncTestSuite.java:27]: same as the annotation style problem

File: hugegraph-client/src/main/java/org/apache/hugegraph/serializer/direct/reuse/BytesDemo.java [hunk 7]
```diff
@@ -124,14 +124,17 @@ private void writeGraphElements() {
         vadasB.property("price", "120");
         vadasB.id(12345);
 
-        Edge peterCreateLop = new Edge("created").source(peter).target(lop).property("date", "2017-03-24");
+        Edge peterCreateLop = new Edge("created").source(peter).target(lop)
+                                                      .property("date", "2017-03-24");
```
[the remark this answers is not in the document]
Comment (imbajin) [reply hugegraph-client/src/main/java/org/apache/hugegraph/serializer/direct/reuse/BytesDemo.java:128]: a little strange
image

**Review by javeme — APPROVED** [review]

**Review by zyxxoo — APPROVED** [review]

**Pull request merged**